test/unit/walk-manifest.spec.js

```diff
@@ -153,8 +160,15 @@ describe('walk-manifest', function() {
         .replyWithFile(200, `${process.cwd()}/test/resources/simple.m3u8`);
 
       var errors = [];
-      var addErrorIntoErrors = function(err, resources, callback) {
-        errors.push(err);
+      var addErrorIntoErrors = function(err, uri, resources, callback) {
```

Comment:
wrapping the object in a closure (similar to the previous example) would avoid the need to duplicate the function